conans/model/version_range.py

```diff
@@ -88,6 +88,8 @@ def _parse_expression(expression):
             if expression[1] == "=":
                 operator += "="
                 index = 2
+        elif expression[1] == "=":
+            raise ConanException(f"Invalid version range operator '{operator}=' in {expression}")
```

Comment:
```suggestion
            raise ConanException(f"Invalid version range operator '{operator}=' in {expression}, you should probably use {operator} instead.")
```
Maybe give a hint to the user on what is the correct range (the proposed error messages doesn't work in all cases, for instance for the `==` one)

Reply:
a better error message would probably be to distinguish between `~=`, `^=`, `==` and the rest.